lib/main_common.pm

```diff
@@ -1634,7 +1634,6 @@ sub load_extra_tests_console {
             loadtest "console/wavpack";
         }
     }
-    loadtest "console/libvorbis";
```

Comment:
This will remove the package also for SLE < 15-SP2.

We should create a YAML schedule for **extra_tests_textmode** in https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/functional
and not schedule **libvorbis** in the YAML schedule.

After that, the test suites in:
- aarch64 https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L115
- ppc64le https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L206
- s390x https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L335
- x86_64 https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L518
- XEN-PV https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L522
- XEN-HVM https://gitlab.suse.de/qsf-u/qa-sle-functional-userspace/-/blob/master/functional_sle15_jobgroup.yaml#L526

need to be changed to **extra_tests_textmode** and use **YAML_SCHEDULE** setting:

```yaml
    - extra_tests_in_textmode:
        settings:
          YAML_SCHEDULE: schedule/functional/extra_tests_textmode.yaml
```

Reply:
So am I understanding right that you want to keep using `extra_tests_in_textmode` with the libvorbis test module for <SLE15-SP2?

Reply:
We only "care" about products under development. But, we should not break tests from other teams.
When all teams have their tests migrated to YAML (if this is what they want) then main.pm can be removed completely.

So, I want to leave extra_tests_in_textmode and main.pm schedule for those teams.

Reply:
Also, remember openSUSE, which doesn't have anything to do with modules.

Reply:
Updated PR.